### PR TITLE
[GEN 2][Mobile] Add missing Mobile code snippets for Set up Amplify Auth.

### PR DESCRIPTION
**Pull request by salihgueler**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [x] Swift
- [x] Android
- [x] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by josefaidt — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 1]
```diff
@@ -352,7 +375,359 @@ The `amplify-authenticator` component offers a simple way to add authentication
...
+    - AWSCognitoAuthPlugin
+
+- Amplify UI Swift - Authenticator: Enter its (GitHub URL)[https://github.com/aws-amplify/amplify-ui-swift-authenticator], *select Up to Next Major Version* and click *Add Package*
+
+  - Select the following library:
+    - Authenticator
+
+Once you are done, add the API dependencies to your project. Select **File > Add Package Dependencies...** and add the `AWSAPIPlugin`.
+
+![Shows the Amplify API library for Swift selected](/images/lib/getting-started/ios/set-up-swift-9.png)
```
Comment (timngyn) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:564]: Could we get a more descriptive alt text for this to better help users who are using screen readers?

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 2]
```diff
@@ -131,18 +131,41 @@ See the [Amazon Cognito documentation](https://docs.aws.amazon.com/cognito/lates
 
 After you have chosen and set up your authentication resource, run the following command to create your first authentication resource in your personal cloud sandbox.
 
+<InlineFilter filters={['flutter']}>
+```bash title="Terminal"
+npx amplify sandbox --config-format dart --config-out-dir lib
+```
+</InlineFilter>
+<InlineFilter filters={['android','angular','javascript','nextjs','react','react-native','vue']}>
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:139]:
```suggestion
<InlineFilter filters={['angular','javascript','nextjs','react','react-native','vue']}>
```

**Review by renebrandel — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 3]
```diff
@@ -352,7 +375,359 @@ The `amplify-authenticator` component offers a simple way to add authentication
...
+  }
+}
+```
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+
+Move the generated files to your project. You can do this by dragging and dropping the files to your project.
+
+![Shows the drag and drop phase](/images/lib/getting-started/ios/set-up-swift-8.png)
```
[the remark this answers is not in the document]
Comment (salihgueler) [reply src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx]: Done ✅

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx (hunk 1 above)
Comment (salihgueler) [reply src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:564]: Done ✅

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 4]
```diff
@@ -131,18 +131,41 @@ See the [Amazon Cognito documentation](https://docs.aws.amazon.com/cognito/lates
...
+```bash title="Terminal"
+npx amplify sandbox --config-format=json-mobile --config-out-dir=app/src/main/res/raw
+```
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+```bash title="Terminal"
+npx amplify sandbox --config-format=json-mobile
+```
+</InlineFilter>
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:159]: add the comment to "Drag & Drop" the file into Xcode

Comment (renebrandel) [reply src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:159]: nvm. you got it below.

**Review by renebrandel — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Pull request merged**